compiler/aarch64/codegen/ControlFlowEvaluator.cpp

```diff
@@ -109,10 +109,11 @@ static TR::Instruction *ificmpHelper(TR::Node *node, TR::ARM64ConditionCode cc,
    TR::LabelSymbol *dstLabel;
    TR::Instruction *result;
    TR::RegisterDependencyConditions *deps;
+   bool isSecondChildNeedsRelocation = cg->profiledPointersRequireRelocation() && (secondChild->getOpCodeValue() == TR::aconst) &&
```

Comment:
I propose an alternative variable name `secondChildNeedsRelocation`.

Reply:
Changed the variable name to `secondChildNeedsRelocation`.